cypress/e2e/1_cluster-welcome.cy.js

```diff
@@ -44,46 +44,13 @@ describe('Welcome page', () => {
     cy.contains('Data container');
     cy.contains('Running'); // cluster status
 
-    //Checks if navigation menu is hidden
-    cy.contains('Data Container').should('not.be.visible');
-    cy.contains('Global Statistics').should('not.be.visible');
-    cy.contains('Cluster Membership').should('not.be.visible');
-
-    cy.get('#nav-toggle').click();
+    cy.get('[data-cy=sideBarToggle]').click();
     //Checks if navigation menu is visible
     cy.contains('Data Container').should('be.visible');
     cy.contains('Global Statistics').should('be.visible');
     cy.contains('Cluster Membership').should('be.visible');
-
-    //Clicks the Cluster Membership link and should go to Cluster Membership page
-    cy.contains('Cluster Membership').click();
-    cy.contains('Cluster membership').should('be.visible');
-    cy.contains('Healthy');
-    cy.contains('infinispan-4-e2e');
-
-    //Clicks the Global statistics link and should go to Global statistics page
-    cy.contains('Global Statistics').click();
-    cy.contains('Global statistics').should('be.visible');
-    cy.contains('Cluster-wide statistics');
-    cy.contains('Cache Manager lifecycle values');
-
-    //Clicks the Access management link and should go to Access management page
-    cy.contains('Access Management').click();
-    cy.contains('Access management').should('be.visible');
-    cy.contains('Access control');
-    cy.contains('Create role');
-
-    //Clicks the Connected clients link and should go to Connected clients page
-    cy.contains('Connected Clients').click();
-    cy.contains('Connected clients').should('be.visible');
-    cy.contains('Client library');
-    cy.contains('Server node');
-
-    //Clicks the Data Container link and should go to Data Container page
-    cy.contains('Data Container').click();
-    cy.contains('Data container').should('be.visible');
-    cy.contains('Running'); // cluster status
-    cy.contains('Cluster rebalancing on'); // rebalancing status
```

Comment:
@karesti is it necessary for the part above to be removed? This execution part is not taking long, but on the other hand verifies that the pages are properly viewed to the user. I remember I have added this part, as after new authentication impl we had an issue that the pages were not properly viewed to the user saying there is no view permission.

Reply:
I think I removed parts that were tested multiple times, but I could do something wrong. Let me check this one for sure @andyuk1986